de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/Negotiation.java

```diff
@@ -70,10 +67,8 @@
    * @param transmitter transmitter used for sending negotiation messages
    * @param receiver receiver used for receiving negotiation messages
    */
-  protected Negotiation(
```

Comment:
I do not get the changes made here. You have an user object in the subclasses which has access to a JID and so there is no reason to make getPeer abstract.

Reply:
Yeah sorry this was based on my misconception that some places would call Negotiation.getPeer(). I mixed that up with something different and now removed the abstract method.

Reply:
You misunderstood me. You should not remove this method. You should change the CTOR super calls in the affected subclasses e.g AbstractProjectNegotiation(User user ....){super(user.getJid()) ...

Your commit message states that you did a "refactor" (this patch here is not a refactor as we use the refactor tag) on the project negotiation classes. This class here is the base class. So either put the code back or amend your commit message.

Reply:
Ok, sorry for the wrong tag.
I changed the title / commit message.